<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raytracing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ImagePainter" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_scene_file"/>
    <addaction name="actionOpen_render_settings"/>
    <addaction name="separator"/>
    <addaction name="actionSave_render_settings"/>
    <addaction name="separator"/>
    <addaction name="actionSave_image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionSelect_view"/>
    <addaction name="actionRender"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInit_camera"/>
    <addaction name="actionRender_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_scene_file"/>
   <addaction name="actionOpen_render_settings"/>
   <addaction name="separator"/>
   <addaction name="actionRender_settings"/>
   <addaction name="actionSave_render_settings"/>
   <addaction name="separator"/>
   <addaction name="actionSave_image"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_view"/>
   <addaction name="actionRender"/>
   <addaction name="separator"/>
   <addaction name="actionInit_camera"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_scene_file">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/open_scene.png</normaloff>:/new/prefix1/icon/open_scene.png</iconset>
   </property>
   <property name="text">
    <string>Open scene file</string>
   </property>
   <property name="statusTip">
    <string>Open scene file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_render_settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/open_render.png</normaloff>:/new/prefix1/icon/open_render.png</iconset>
   </property>
   <property name="text">
    <string>Open render file</string>
   </property>
   <property name="statusTip">
    <string>Open render file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/084_photo.png</normaloff>:/new/prefix1/icon/084_photo.png</iconset>
   </property>
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="statusTip">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_render_settings">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/279a2c8bc7a76ce437cace378e4570fa.png</normaloff>:/new/prefix1/icon/279a2c8bc7a76ce437cace378e4570fa.png</iconset>
   </property>
   <property name="text">
    <string>Save render settings</string>
   </property>
   <property name="statusTip">
    <string>Save render settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSelect_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/eye.png</normaloff>:/new/prefix1/icon/eye.png</iconset>
   </property>
   <property name="text">
    <string>Select view</string>
   </property>
   <property name="statusTip">
    <string>Select view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/yelp.png</normaloff>:/new/prefix1/icon/yelp.png</iconset>
   </property>
   <property name="text">
    <string>Render</string>
   </property>
   <property name="statusTip">
    <string>Render</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInit_camera">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/clear.png</normaloff>:/new/prefix1/icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>Init camera</string>
   </property>
   <property name="statusTip">
    <string>Init camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRender_settings">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/32px-Write-icon.svg.png</normaloff>:/new/prefix1/icon/32px-Write-icon.svg.png</iconset>
   </property>
   <property name="text">
    <string>Render settings</string>
   </property>
   <property name="statusTip">
    <string>Render settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Return</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icon/help.png</normaloff>:/new/prefix1/icon/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImagePainter</class>
   <extends>QWidget</extends>
   <header location="global">gui/imagepainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
